client/css/ProductOverview.scss

```diff
@@ -47,33 +64,51 @@
 .socialMedia {
   display: flex;
   margin: 10px 0;
-  justify-content: left;
-}
+  justify-content: space-between;
+  width: 50%;
 
-.styleIndicator {
-  font-weight: bold;
-}
+  svg {
+    transition: color 200ms;
+  }
 
-.styleSelected {
-  font-size: 25px;
+  svg:hover {
+    color: goldenrod;
+    transition: color 200ms;
+  }
 }
 
 .productDescription {
-  border: 2px solid black;
-  margin: 1% 0;
-  font-size: 25px;
+  color: #777;
+  display: flex;
+  padding: 25px 0;
+
+  &__info {
+    border-right: 2px solid #bbb;
+    padding: 20px 45px;
+    flex: 2;
```

Comment:
I would like to learn more about this flex option. To my understanding, the number dictates how the element will the fill the space but I am still unsure of the exact details on how it fills the space.

Reply:
In this situation, we have two flex items (info and features) which are the two children of productDescription. Setting info to flex: 2 and features to flex: 1 means that info will be twice the size as one or 66.6% vs 33.3%. If we set info to flex: 3 then it would take up 75% and features would have 25%. https://css-tricks.com/snippets/css/a-guide-to-flexbox/